src/etc/inc/plugins.inc.d/ipsec.inc

```diff
@@ -1,6 +1,8 @@
 <?php
 
 /*
+ * Copyright (C) 2021 Jasper Roloff
+ * Copyright (C) 2021 Jürgen Walch
```

Comment:
This really does not reach the threshold for originality so copyright claims are void.

Reply:
Oh, sorry – I saw other issues where the creators were asked to add a copyright notice.
Nevermind, I've reverted that to the original state now.